docs/_posts/2017-02-11-warts.md

```diff
@@ -47,6 +47,15 @@ val xs = List(1, true)
 x.asInstanceOf[String]
 ```
 
+### Contains
+`contains` in Lists and Options is type unsafe it is possible to implement List(1).contains("1"). Use type-safe implementation for example from cats `List.contains_`
+````scala
+// Won't compile: List.contains is disabled
+List(1).contains(1)
```

Comment:
but _this_ is the useful case, isn't it? it might make sense to only disable them for un-related types...

Reply:
that is generally a bad idea to use native _contains_ , my opinion is that always something like cats.contains_ should be used to be safe